src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -249,9 +249,31 @@ public async Task<IEnumerable<WorkloadDownload>> GetManifestPackageDownloadsAsyn
             {
                 try
                 {
+                    
```

Comment:
We should update `UpdateAdvertisingManifestAsync` so that it calls `GetManifestPackageDownloadsAsync` instead of having the logic duplicated.